orchagent/main.cpp

```diff
@@ -529,14 +528,14 @@ int main(int argc, char **argv)
 
     // Instantiate ZMQ server
     shared_ptr<ZmqServer> zmq_server = nullptr;
-    if (enable_zmq)
+    if (zmq_server_address.empty())
     {
-        SWSS_LOG_NOTICE("Instantiate ZMQ server : %s, %s", zmq_server_address.c_str(), vrf.c_str());
-        zmq_server = make_shared<ZmqServer>(zmq_server_address.c_str(), vrf.c_str());
+        SWSS_LOG_NOTICE("ZMQ disabled");
```

Comment:
Could you update the log message as well specifying which zmq channel is disabled?

Reply:
Fixed